<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="7" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-15T16:35:43 IST" name="Suite" finished-at="2025-03-15T16:36:23 IST" duration-ms="39373">
    <groups>
    </groups>
    <test started-at="2025-03-15T16:35:43 IST" name="basic tests" finished-at="2025-03-15T16:36:23 IST" duration-ms="39373">
      <class name="testCases.Mainmaster_hoidayTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.Mainmaster_hoidayTest@74e28667]" started-at="2025-03-15T16:36:07 IST" name="setup" finished-at="2025-03-15T16:36:19 IST" duration-ms="12786" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateTheHoliBtnTest()[pri:0, instance:testCases.Mainmaster_hoidayTest@74e28667]" started-at="2025-03-15T16:36:19 IST" name="validateTheHoliBtnTest" finished-at="2025-03-15T16:36:22 IST" duration-ms="3035" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTheHoliBtnTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.Mainmaster_hoidayTest@74e28667]" started-at="2025-03-15T16:36:22 IST" name="tearDown" finished-at="2025-03-15T16:36:23 IST" duration-ms="335" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.Mainmaster_hoidayTest -->
      <class name="testCases.LoginTest">
        <test-method is-config="true" signature="loadConfig()[pri:0, instance:testCases.LoginTest@7ef82753]" started-at="2025-03-15T16:35:42 IST" name="loadConfig" finished-at="2025-03-15T16:35:43 IST" duration-ms="1113" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadConfig -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.LoginTest@7ef82753]" started-at="2025-03-15T16:35:43 IST" name="setup" finished-at="2025-03-15T16:35:50 IST" duration-ms="6178" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest()[pri:0, instance:testCases.LoginTest@7ef82753]" started-at="2025-03-15T16:35:50 IST" name="loginTest" finished-at="2025-03-15T16:35:53 IST" duration-ms="3569" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.LoginTest@7ef82753]" started-at="2025-03-15T16:35:53 IST" name="tearDown" finished-at="2025-03-15T16:35:54 IST" duration-ms="343" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="CreateReport()[pri:0, instance:testCases.LoginTest@7ef82753]" started-at="2025-03-15T16:36:23 IST" name="CreateReport" finished-at="2025-03-15T16:36:23 IST" duration-ms="382" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateReport -->
      </class> <!-- testCases.LoginTest -->
      <class name="testCases.HolidayTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.HolidayTest@7161d8d1]" started-at="2025-03-15T16:35:54 IST" name="setup" finished-at="2025-03-15T16:36:05 IST" duration-ms="11365" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="HolidayCreationTest()[pri:0, instance:testCases.HolidayTest@7161d8d1]" started-at="2025-03-15T16:36:05 IST" name="HolidayCreationTest" finished-at="2025-03-15T16:36:05 IST" duration-ms="577" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at testCases.HolidayTest.HolidayCreationTest(HolidayTest.java:54)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HolidayCreationTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.HolidayTest@7161d8d1]" started-at="2025-03-15T16:36:06 IST" name="tearDown" finished-at="2025-03-15T16:36:07 IST" duration-ms="909" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.HolidayTest -->
    </test> <!-- basic tests -->
  </suite> <!-- Suite -->
</testng-results>
